localstack/services/edge.py

```diff
@@ -204,6 +206,16 @@ def return_response(self, method, path, data, headers, response):
                     response.content,
                 )
 
+        # convert XML responses to JSON - TODO: introduce generalizable response conversion
+        if headers.get("Accept") == "application/json":
+            try:
+                # simple heuristic to detect XML responses
+                if str(response._content or "").startswith("<"):
+                    content = xmltodict.parse(to_str(response._content))
+                    response._content = strip_xmlns(content)
+            except Exception as e:
+                LOG.debug("Unable to convert XML response to JSON: %s", e)
```

Comment:
```suggestion
                LOG.debug("Unable to convert XML response to JSON", exc_info=e)
```

Depends on what's the strategy with this going forward. If we want to keep this from cluttering the logs we can also leave it as is, but otherwise it might make sense to print the traceback.